docs/docs/getting-started/install.md

```diff
@@ -291,6 +291,12 @@ The built-in `PowerUserAccess` policy is a superset of these permissions.
 
```

Comment:
Add STACKIT to the list of supported CSPs under prerequisites at the beginning of this page.
Also add it to the analog lists on marketplaces.md and ../overview/license.md